components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -1751,6 +1751,20 @@ def seppuku(self, softclose=False):  # pragma: no cover
         session is really terminate disregarding its connection refcount.
 
         :param softclose:   Boolean
+
+        ** Deprecated ** Use :meth:`terminateAllClients`.
+        Our apologies for any offense caused by this previous method name.
+        """
+        warnings.warn("Deprecated. Use terminateAllClients",
+                      DeprecationWarning)
+        self.terminateAllClients(softclose)
+
+    def terminateAllClients(self, softclose=False):  # pragma: no cover
+        """
+        Terminates connection with killSession(). If softclose is False, the
+        session is really terminate disregarding its connection refcount.
```

Comment:
termintate -> terminated
Actually, this whole doc string could do with clarifying as I've never really understood what's going on here.
Even now, looking at the code, the logic is not really clear to me.
```terminateAllClients()``` seems a little strong. What about ```terminateClients()``` since if softClose is True then you don't terminate all?